OtripleS.Web.Api.Tests.Unit/Services/Foundations/GuardianContacts/GuardianContactServiceTests.Validations.RetrieveById.cs

```diff
@@ -18,21 +18,19 @@ public partial class GuardianContactServiceTests
         public async Task ShouldThrowValidatonExceptionOnRetrieveWhenGuardianIdIsInvalidAndLogItAsync()
         {
             // given
-            Guid randomContactId = Guid.NewGuid();
+            Guid invalidContactId = Guid.NewGuid();
```

Comment:
If you scope is testing invalidContactId then you need the problem details as well.